web/pages/Settings/index.tsx

```diff
@@ -35,22 +29,16 @@ const adapterOptions = CHAT_ADAPTERS.filter((adp) => adp !== 'default') as Defau
 
 const Settings: Component = () => {
   const state = userStore()
-  const cfg = settingStore()
 
-  createEffect(() => {
-    refreshHorde()
-  })
-
-  const [workers, setWorkers] = createSignal<Option[]>()
-  const [show, setShow] = createSignal(false)
   const [tab, setTab] = createSignal(0)
 
-  const tabs = ['ai', 'ui', 'service'] satisfies Tab[]
+  const tabs: Tab[] = ['ai', 'ui']
+  if(!state.loggedIn) tabs.push('guest')
```

Comment:
I'd highly recommend running using the Prettier as your default formatter and configure your IDE to format on save. 
In the mean time run `pnpm format:fix` before pushing.